po/es.po

```diff
@@ -0,0 +1,144 @@
+# Spanish translations for com.github.elfenware.obliviate package.
+# Copyright (C) 2020 THE com.github.elfenware.obliviate'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.elfenware.obliviate package.
+# Darshak Parikh, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.elfenware.obliviate\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-12-18 20:37+0530\n"
+"PO-Revision-Date: 2020-12-18 20:37+0530\n"
+"Last-Translator: Darshak Parikh\n"
+"Language-Team: Español\n"
+"Language: es\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: src/MainView.vala:43 src/MainView.vala:160
+msgid "Copied to clipboard"
+msgstr "Copiado al portapapeles"
+
+#: src/MainView.vala:48
+msgid "Site:"
+msgstr "Sitio:"
```

Comment:
Even though this syntactically correct, we usually refer to a web page as "Página web" or just "Web".

```suggestion
msgstr "Página web:"
```

or

```suggestion
msgstr "Web:"
```